libs/ledger-live-common/src/api/Ethereum.ts

```diff
@@ -340,16 +361,15 @@ export const apiForCurrency = (currency: CryptoCurrency): API => {
       }
     ),
 
-    async getBlockByHash(blockHash) {
+    async getBlockByHash(blockHash): Promise<BlockByHashOutput | undefined> {
```

Comment:
can we type `blockByHash` and make it mandatory ? When does it happen that `blockHash` can be undefined ?

Reply:
I think it's because you can try to look for a block that doesn't exist, it's a search